pgjdbc/src/main/java/org/postgresql/util/StreamWrapper.java

```diff
@@ -23,26 +23,37 @@
  *
  * @author Oliver Jowett (oliver@opencloud.com)
  */
-public class StreamWrapper implements Closeable {
+public final class StreamWrapper implements Closeable {
 
   private static final int MAX_MEMORY_BUFFER_BYTES = 51200;
 
   private static final String TEMP_FILE_PREFIX = "postgres-pgjdbc-stream";
 
+  private LazyCleaner.Cleanable cleaner;
+
+  @SuppressWarnings("initialization")
   public StreamWrapper(byte[] data, int offset, int length) {
     this.stream = null;
     this.rawData = data;
     this.offset = offset;
     this.length = length;
+    // create a null lambda to avoid checker errors
+    finalizeAction = (LazyCleaner.CleaningAction)(boolean clean) -> { };
+    cleaner = LazyCleaner.getInstance().register(this, finalizeAction);
```

Comment:
It might be slightly cleaner to use a temp variable to keep suppression closer to the offending statement:

```java
@SuppressWarnings("initialization")
Cleanable cleanable = LazyCleaner.getInstance().register(this, finalizeAction);
this.cleanable = cleanable;
```

Reply:
still doesn't stop checker from complaining about it.